Assets/PatchKit Patcher/Scripts/AppStarter.cs

```diff
@@ -26,12 +27,39 @@ public AppStarter(App app)
             AppFinder = new AppFinder();
         }
 
+        private string ResolveExecutablePath(AppVersion appVersion)
+        {
+            if (!string.IsNullOrEmpty(appVersion.MainExecutable))
+            {
+                string executablePath = Path.Combine(_app.LocalDirectory.Path, appVersion.MainExecutable);
+
+                if (File.Exists(executablePath))
+                {
+                    return executablePath;
+                }
+
+                // Reports to Sentry
+                DebugLogger.LogException(
+                    new FileNotFoundException(string.Format("Couldn't resolve executable in {0}", executablePath)));
```

Comment:
If you create an exception you should always throw it. Otherwise stack trace information is not filled inside it.

So better solution for that would be to wrap this code inside `try` and report exception to Sentry in `catch`.